README.md

```diff
@@ -7,5 +7,6 @@ a new brach and add your name and send a pull request. Your entry should
 maintain the alphabetic order and it should be in the format `your name <email
 id>`.
 
-Aditya Narayan <narayanaditya95@gmail.com>  
+Abinash Meher <abinashdakshana999@gmail.com><br/>
```

Comment:
That would make the next entry a new paragraph. Are you sure you want that?

*Regards,*
*Abinash Meher*

On Thu, Feb 26, 2015 at 12:41 AM, Harsh Gupta <notifications@github.com>
wrote:

> In README.md
> <https://github.com/AGV-IIT-KGP/git-sandbox/pull/7#discussion_r25368812>:
>
> > @@ -7,5 +7,6 @@ a new brach and add your name and send a pull request. Your entry should
> >  maintain the alphabetic order and it should be in the format `your name <email
> >  id>`.
> >
> > -Aditya Narayan <narayanaditya95@gmail.com>
> > +Abinash Meher <abinashdakshana999@gmail.com><br/>
>
> You should add a new empty line instead of adding <br/>, also correct it
> for Aditya's entry. Have a new line after every line with name.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/AGV-IIT-KGP/git-sandbox/pull/7/files#r25368812>.
>